docs/src/validator/tpu.md

```diff
@@ -9,12 +9,20 @@ responsible for block production.
 
 ![TPU Block Diagram](/img/tpu.svg)
 
-Transactions encoded and sent in UDP packets flow into the validator
+Transactions are encoded and sent in QUIC streams into the validator
 from clients (other validators/users of the network) as follows:
 
-* fetch stage: allocates packet memory and reads the packet data from
-the network socket and applies some coalescing of packets received at
-the same time.
+* The quic streamer: allocates packet memory and reads the packet data from
+the QUIC endpoint and applies some coalescing of packets received at
+the same time. Each stream is used to transmit a packet. There is a limit on
+the maximum streams can be concurrently opened per (source IP address,
```

Comment:
I think it's either IP address, or Pubkey. For unstaked connections we are using IP to identify a connection. For staked, we are using the Pubkey.